liquidprompt

```diff
@@ -948,14 +951,34 @@ _lp_git_branch_color()
                 d_lines=0
             fi
 
-            local has_lines
-            has_lines="+$i_lines/-$d_lines"
+            diff_details+="+$i_lines/-$d_lines"
+        fi
+
+        local status_details=""
+        if (( LP_ENABLE_GIT_STATUS )); then
+            local status_lines="$(LC_ALL=C \git status --porcelain 2>/dev/null)"
+
+            if [[ -n "$status_lines" ]]; then
+                local staged="$(echo "${status_lines}" | \grep '^[^? ]. .*' | wc -l)"
```

Comment:
Let's make this:
```local staged="$(echo "${status_lines}" | \grep '^[^? ]. .*' | wc -l | sed 's/ //g')"```

On OSX there's a ton of spaces that match as well. This will trim them out.

Reply:
fix pushed